snapcraft.yaml

```diff
@@ -58,8 +60,8 @@ parts:
       'bin/console-conf-tui': usr/bin/console-conf
       'bin/subiquity-tui': usr/bin/subiquity
       'bin/subiquity-loadkeys': usr/bin/subiquity-loadkeys
-      'bin/curtin-journald-forwarder': usr/bin/curtin-journald-forwarder
```

Comment:
@mwhudson why is that getting deleted? I thought we use it? or not any more?

Reply:
the actual file referenced here got deleted a year ago, when curtin implemented reporting to the journal natively. One might have expected snapcraft to complain about me forgetting to delete it here too but well. Apparently it didn't!